tests/test_launch.py

```diff
@@ -20,9 +22,21 @@ def setUp(self):
         # Set up the schema
         root_repo_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
         self.template_dir = os.path.join(root_repo_dir, "nf_core", "pipeline-template")
-        self.nf_params_fn = os.path.join(tempfile.mkdtemp(), "nf-params.json")
+        # cannot use a context manager here, since outside setUp the temporary
+        # file will never exists
+        self.tmp_dir = tempfile.mkdtemp()
```

Comment:
For these tests, I need the temporary folder. Track it in a class attribute. Will be removed in the `tearDown` function after each test